docs/guides/soloproject/soloproject.md

```diff
@@ -44,10 +50,11 @@ intermediate (Tier 2), or experienced (Tier 3).
 <tr>
 <td>Tier&nbsp;1</td>
 <td>
-<p>This tier is for developers who are just starting out in web development.</p>
+<p>This tier is for developers who are starting out in frontend web development.</p>
 <strong>Skills:</strong>
 <ul>
     <li>Basic understanding of CSS and HTML.</li>
+    <li>Ability to create simple apps using any frontend framework such as ReactJS, SvelteJS, VueJS, Angular, Django, etc.
```

Comment:
Isn't this a little misleading? Further down it says tier 1 is only CSS, HTML and a programming language. framework is tier 2

Reply:
**UPDATE @ellrub ** I re-read your comment and have pushed a new commit to fix the above. Sorry for not catching this correctly.

Reply:
Perfect